bundle/manifests/maintenance-operator-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/component: kube-rbac-proxy
```

Comment:
Deprecated
https://github.com/Mellanox/network-operator/issues/1201

Reply:
Dropped it, thanks!